packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -38,6 +38,8 @@ class SystemRulesTable extends React.Component {
         itemsPerPage: this.props.itemsPerPage,
         rows: [],
         hidePassed: this.props.hidePassed,
+        selectedFilter: this.props.selectedFilter,
+        selectedRefIds: this.props.selectedRefIds,
```

Comment:
There are *many occasions* where the `rows` or `currentRows` will be empty arrays or will just not contain all of the rules. For example, if you search for something that doesn't exist, or if you select some severity filter.

Because of that, we cannot keep the state for the selected rows in neither `currentRows`, or `rows`. Instead, I'm keeping an array of refIds that are selected, which is fairly easy to update, and also to send to redux-form.